R/vdiagram-render.R

```diff
@@ -67,7 +67,10 @@ createVdiagram <- function(data) {
 
   vplot <- testCallouts(vplot, xlimits = xlim(vplot)$side.1)
   
-  ylims <- ylim(vplot)$side.2
+  ylims <- c(
+      min(c(ylim(vplot)$side.2, vdiagramData$minStage)),
+      max(c(ylim(vplot)$side.2, vdiagramData$maxStage))
+      )
```

Comment:
also, while you're changing this stuff, can you change the axis call to be just an axis call rather than inside a do.call. `do.call` is only needed when you have the list of styles to add or in a loop.

This should work just fine:
```r
vplot <- axis(vplot, side = c(1, 2, 4), at = c(x_seq, y_seq, y_seq)) %>% 
    view(side=2, ylim=ylims)
```